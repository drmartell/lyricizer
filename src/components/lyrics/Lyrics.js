import React from 'react';
import useLyrics from '../../hooks/useLyrics.js';
import queryString from 'query-string';
import PropTypes from 'prop-types';

export default function Lyrics({ location }) {
  const { artist, title } = queryString.parse(location.search);
  const { lyrics, loading } = useLyrics(artist, title);


  return (
    <p>
      {lyrics}
    </p>
  );
}
