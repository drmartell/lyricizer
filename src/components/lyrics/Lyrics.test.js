import React from 'react';
import { shallow } from 'enzyme';
import Lyrics from './Lyrics';

describe('Lyrics component', () => {
  it('should render Lyrics', () => {
    const wrapper = shallow(<Lyrics location={{ search: 'hi' }} />);
    expect(wrapper).toMatchSnapshot();
  });
});
