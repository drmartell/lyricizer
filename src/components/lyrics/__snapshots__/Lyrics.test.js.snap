// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lyrics component should render Lyrics 1`] = `<pre />`;
